plugins/processors/k8sdecorator/stores/utils_test.go

```diff
@@ -14,7 +16,14 @@ func TestUtils_parseDeploymentFromReplicaSet(t *testing.T) {
 }
 
 func TestUtils_parseCronJobFromJob(t *testing.T) {
-	assert.Equal(t, "", parseCronJobFromJob("hello-123"))
-	assert.Equal(t, "hello", parseCronJobFromJob("hello-1234567890"))
-	assert.Equal(t, "", parseCronJobFromJob("hello-123456789a"))
+	//For CronJobControllV2 which is after K8s v1.21
+	assert.Equal(t, "hello", parseCronJobFromJob("hello-"+strconv.FormatInt(time.Now().Unix()/60, 10)))
+	assert.Equal(t, "", parseCronJobFromJob("hello-"+strconv.FormatInt(time.Now().Unix()/60, 10)+"abc"))
+	assert.Equal(t, "", parseCronJobFromJob("hello-"+strconv.FormatInt(time.Now().Unix()/60, 10)+"-name"))
+
+	//For CronJob which is before K8s v1.21
+	assert.Equal(t, "hello", parseCronJobFromJob("hello-"+strconv.FormatInt(time.Now().Unix(), 10)))
```

Comment:
Do the original test assertions not work? If not, then this looks like a breaking change.

Reply:
The original test assertions only covers the Cron Job but not CronJobControllerV2. Moreover, it does not cover some test case such as parsing Unix Time to integer with special characters though.